src/models/Factor.js

```diff
@@ -207,7 +207,7 @@ function (Okta, Q, factorUtil, BaseLoginModel) {
         }
 
         // MFA_CHALLENGE
-        else if (this.get('canUseResend') && transaction.resend) {
+        else if (this.get('canUseResend') && !this.get('answer') && transaction.resend) {
```

Comment:
@mauriciocastillosilva-okta 
Could you explain what do you mean?
The main logic is in the model, so I don't understand what to change in the Form.
Moving `resend` to the specific function is not a good idea since PushForm also uses it by calling `save`, so this way there will be a duplicated logic.